src/violations.rs

```diff
@@ -0,0 +1,5246 @@
+use itertools::Itertools;
```

Comment:
We should do some light reformatting of this file prior to merging: put all the section headers (`// pycodestyle errors`) on their own lines, add a blank line before each `define_violation!(...)`, etc.

Reply:
Good point ... I can adapt the Python script of the third commit to take care of that.